test/functional/host_test.rb

```diff
@@ -219,3 +219,29 @@
     result.exit_code.must_equal HammerCLI::EX_OK
   end
 end
+describe 'disassociate host from vm' do
+  let(:cmd) { ["host","disassociate"] }
+
+  it 'successful disassociate host' do
+    params = ['--id=1']
+    expected_result = success_result("The host has been disassociated from VM\n")
+    api_expects(:hosts, :disassociate, 'disassociate hosts') do |params|
+      params['id'] == "1"
+    end.returns()
```

Comment:
Please give it some return value. `returns({})` should be enough in this case. It will make the return value more obvious. I'm not sure what's the behavior of `returns` without any value.